netutils/lib_helpers.py

```diff
@@ -13,6 +14,31 @@
 else:
     HAS_NAPALM = True
 
+UPGRADE_PATHS = {
```

Comment:
Do we want to keep track of this or should this be up to each user to track?  Seem like it'd require a lot of maintenance.

Reply:
You're probably right about this. The function works entirely with externally provided lists of version numbers, and these were convenient extras for the test cases I was building this around.